lib/scavenger_hunt/app/views/scavenger_hunt/answers/_form.html.slim

```diff
@@ -3,7 +3,7 @@
   = text_field_tag :answer, "", placeholder: "Enter object title"
 
     = toolbar_item do
-      = link_to "Return to clue", game_clue_path(@game, @clue), class: "large"
+      = link_to "Return<br />to clue".html_safe, game_clue_path(@game, @clue), class: "large"
```

Comment:
Just a nifty trick to note for future use: you can provide a block to `link_to` instead of hand-writing HTML (generally considered a Bad Idea™). Example:

```
link_to game_clue_path(whatever), class: "whatever" do
  | Return
  br
  | to clue
```

Reply:
I love this! Way cleaner. I just went in and updated all of the places the code looked like this!